ubidots/ubidots_in.html

```diff
@@ -1,111 +1,608 @@
-        </label>
-        <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
-    </div>
+      <div class="form-row">
+          <label for="node-input-name">
+            <i class="fa fa-tag"></i>
+        <span data-i18n="ubidots.name"></span>
+    </label>
+          <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
+      </div>
 
-    <div class="form-row">
-        <label for="node-input-token">
-            <i class="icon-lock"></i>
-            <span data-i18n="ubidots.token.label"></span>
-        </label>
-        <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
-    </div>
+      <div class="form-row">
+          <label for="node-input-token">
+        <i class="fa fa-lock"></i>
+        <span data-i18n="ubidots.token.label"></span>
+    </label>
+          <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
+      </div>
 
-    <div class="form-row">
-        <label for="node-input-device_label">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.device_label.label"></span>
-        </label>
-        <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
+      <div class="form-row">
+          <label for="node-input-device_label">
+        <i class="fa fa-tag"></i>
+        <span data-i18n="ubidots.device_label.label"></span>
+    </label>
+          <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
+      </div>
+      <div class="form-row">
+        <input type="checkbox" checked id="node-input-tls_checkbox_in" style="display: inline-block; width: auto; vertical-align: top;"></label>
+        <label for="node-input-tls_checkbox_in" style="width: calc(100% - 170px); margin-left:15px" data-i18n="ubidots.tls.label"></label>
     </div>
-
     <div class="form-row">
-        <label for="node-input-label_variable">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.label_variable.label"></span>
-        </label>
-        <input type="text" id="node-input-label_variable" data-i18n="[placeholder]ubidots.label_variable.placeholder" />
-    </div>
-</script>
+      <input type="checkbox" id="node-input-custom_topic_checkbox" style="display: inline-block; width: auto; vertical-align: top;"></label>
+      <label for="node-input-custom_topic_checkbox" style="width: calc(100% - 170px); margin-left:15px" data-i18n="ubidots.custom_topics"></label>
+  </div>
+  <hr>
+
+      <!-- DELETE/SELECT ALL BUTTON -->
+      <div class="form-row">
+        <button id="select-all-variables-button" class="deleteButton" data-i18n="ubidots.variables.selectAll"></button>
+        <button id="delete-variables-button" style="margin-left:210px" class="deleteButton" data-i18n="ubidots.variables.delete"></button>
```

Comment:
I'm not a fan of the `margin-left` in this node. When you finish all the comments, let's review this.